.circleci/config.yml

```diff
@@ -51,16 +51,13 @@ jobs:
             ./compose.sh tox down
 
       - run:
-          name: Tag production container
-          command: |
-            docker tag production:latest eu.gcr.io/$GCLOUD_PROJECT/$CIRCLE_PROJECT_REPONAME:sha-$CIRCLE_SHA1
-            docker tag production:latest eu.gcr.io/$GCLOUD_PROJECT/$CIRCLE_PROJECT_REPONAME:branch-$CIRCLE_BRANCH
-            docker tag production:latest eu.gcr.io/$GCLOUD_PROJECT/$CIRCLE_PROJECT_REPONAME:build-$CIRCLE_BUILD_NUM
-
-      - run:
-          name: Push docker image
+          name: Tag & Push docker image
           command: |
             if [ ! -z "$GCLOUD_SERVICE_KEY" ]; then
+              docker tag production:latest eu.gcr.io/$GCLOUD_PROJECT/$CIRCLE_PROJECT_REPONAME:sha-$CIRCLE_SHA1
+              docker tag production:latest eu.gcr.io/$GCLOUD_PROJECT/$CIRCLE_PROJECT_REPONAME:branch-$CIRCLE_BRANCH
+              docker tag production:latest eu.gcr.io/$GCLOUD_PROJECT/$CIRCLE_PROJECT_REPONAME:build-$CIRCLE_BUILD_NUM
+
```

Comment:
Was this related to #95? If so, I think this is avoid the issue rather than fixing it. I think the "correct" fix is removing the ``branch-...`` tag as outlined in that issue.